### DISCUSSION TITLE
A Railway Template for Open WebUI and Pipelines

### DISCUSSION THREAD

**Discussion by monotykamary** [open-webui/open-webui#6135] [Show and tell]
This is a template I've made for Open WebUI and Pipelines that we run internally at [Dwarves](https://dwarves.foundation/):

https://railway.app/template/Hez7Hu?referralCode=ZqgrJ0

<img width="1390" alt="image" src="https://github.com/user-attachments/assets/0be0e5b4-0c01-4c74-96e9-f975d2280682">

Works with most functions, pipelines (like [o1-reasoning](https://github.com/Tavernari/openwebui-pipelines/blob/main/o1-reasoning.py)), and tools. Only thing I haven't figured it is the [run-code tool](https://openwebui.com/f/etienneperot/run_code)
to run gVisor sandboxed code on it (yet!).

<img width="1582" alt="image" src="https://github.com/user-attachments/assets/6e94cd95-3414-4c45-a0f8-80a2ee110c32">

**Comment by scooter7** [comment]
Hi, do you plan on updating to the most recent version of open webui? Thanks!

**Reply by monotykamary** [reply]
The template pulls packages labelled `main`, so they are technically always the latest. If you have an existing template running on Railway, easiest way to update them is to redeploy each container.

**Comment by thewasta** [comment]
Hello there! 
Anyone knows minimum requirements? Or estimate pricing?
For example do I need pro plan? Or hobby is OK?

Thanks!

**Reply by monotykamary** [reply]
Hobby is ok, I have one hosted in my hobby and one of my team pro plans. It's around $7+ dollars if you're using it on your own. Included usage will cover most of the cost:
<img width="670" alt="image" src="https://github.com/user-attachments/assets/75fd95d4-4413-4204-a9b8-201b1f42b671" />

Can reach up to $15+ if you're using it with a lot of other people (this one is on the pro plan):
<img width="907" alt="image" src="https://github.com/user-attachments/assets/3ed7b7c1-b8a7-4aed-b03f-73eaff01092a" />
